app/controllers/insured/group_selection_controller.rb

```diff
@@ -261,15 +283,19 @@ def edit_aptc
     aptc_applied_total = revise_aptc_applied_total(params, enrollment_id)
     applied_aptc_pct = calculate_elected_aptc_pct(aptc_applied_total.to_f, params[:max_aptc].to_f)
     attrs = {enrollment_id: enrollment_id, elected_aptc_pct: applied_aptc_pct, aptc_applied_total: aptc_applied_total}
-
     begin
       message = ::Insured::Forms::SelfTermOrCancelForm.for_aptc_update_post(attrs)
```

Comment:
a little trivial but this probably doesn't need to be declared, think you could just move it down to the else and say `flash[:notice] = ::Insured...`

Reply:
So that `for_aptc_update_post` actually makes the calls all the way to the `update_aptc` method in SelfServiceFactory, but I modified it a bit and hopefully it looks better:
https://github.com/ideacrew/enroll/pull/4327/commits/adbbf95c4afa9407a11ed2e2d1a124d8ff98d22e#diff-9d1743baad3e5f99189e0df2ac697de176aa1765b4c998afc5429bc1c2570828R288 && https://github.com/ideacrew/enroll/pull/4327/commits/adbbf95c4afa9407a11ed2e2d1a124d8ff98d22e#diff-3492d49e338b8946cd93580ef6c3755f1ff1bd0e28a8e2241e749865408291d2R61-R70

Reply:
Ended up swapping back; that file could not recognize l10n even with the l10nHelper.